BasicSample/Assets/Shared/Scripts/MixedRealityProjectSelectionWindow.cs

```diff
@@ -0,0 +1,195 @@
+        private static PopupUserSettings UserSettings;
+        private const string SettingsFileName = "MixedRealityOpenXRProjectSelectionSettings.asset";
+        private static string UserSettingsFolder => Path.Combine(Application.dataPath, "..", "UserSettings");
+        private static string SettingsAssetPath => Path.Combine(UserSettingsFolder, SettingsFileName);
+
+        [MenuItem("Mixed Reality/Quick Setup", false, 499)]
+        private static void ShowWindowFromMenu()
+        {
+            ShowfromMenu = true;
+            ShowWindow();
+        }
+
+        public static void ShowWindow()
+        {
+            GetUserSettings();
+            // There should be only one configurator window open as a "pop-up". If already open, then just force focus on our instance
+            if (IsOpen)
+            {
+                Instance.Focus();
+            }
+            else
+            {
+                if(!UserSettings.DisablePopup || ShowfromMenu)
+                {
+                    var window = CreateInstance<MixedRealityProjectSelectionWindow>();
+                    window.titleContent = new GUIContent("MixedReality Project Selection Window", EditorGUIUtility.IconContent("_Popup").image);
+                    window.position = new Rect(Screen.width / 2.0f, Screen.height / 2.0f, Default_Window_Height, Default_Window_Width);
+                    window.ShowUtility();
+                }
+
+            }
+        }
+
+        internal static void GetUserSettings()
+        {
+            if(UserSettings == null)
+            {
+                UserSettings = CreateInstance<PopupUserSettings>();
+
+                if (File.Exists(SettingsAssetPath))
+                {
+                    using (StreamReader settingsReader = new StreamReader(SettingsAssetPath))
+                    {
+                        JsonUtility.FromJsonOverwrite(settingsReader.ReadToEnd(), UserSettings);
+                    }
+                }
+                else
+                {
+#pragma warning disable CS0618 // to use the obsolete fields to port to the new asset file
+                    UserSettings.DisablePopup = false;
+#pragma warning restore CS0618
+                }
+            }
+        }
+
+        private void SaveSettings()
+        {
+            if (UserSettings == null)
+            {
+                return;
+            }
+            
+            if (!Directory.Exists(UserSettingsFolder))
+            {
+                Directory.CreateDirectory(UserSettingsFolder);
+            }
+
+            using (StreamWriter settingsWriter = new StreamWriter(SettingsAssetPath))
+            {
+                settingsWriter.Write(JsonUtility.ToJson(UserSettings, true));
+            }
+        }
+
+        private void OnGUI()
+        {
+            EditorGUIUtility.labelWidth = 400;
+            GUILayout.TextField("Change the Unity project settings for Mixed Reality Scenario",GUILayout.Width(350));
```

Comment:
This is the first thing someone opening this project will see - maybe we could add a greeting? :)

![image](https://user-images.githubusercontent.com/12822425/164567295-15b8e491-6914-4805-a4d8-12103e8d6c57.png)

Also, using GUILayout.Label instead of textfield can clean up the look a bit.
```suggestion
            EditorGUIUtility.labelWidth = 400;
            GUILayout.Space(15);
            GUIStyle titleStyle = new GUIStyle(EditorStyles.largeLabel) { fontStyle = FontStyle.Bold };
            GUILayout.Label("Welcome to the Mixed Reality OpenXR Samples!", titleStyle, GUILayout.Width(400));
            GUILayout.Space(10);
            GUILayout.Label("Change this project's settings for your Mixed Reality scenario:", GUILayout.Width(400));
            GUILayout.Space(5);
```

Reply:
thanks for the suggestions! newbie with GUI stuff.

Reply:
I tried things from the Unity https://docs.unity3d.com/ScriptReference/GUILayout.html docs until it looked right; for the style changes, I just reference how MRTK's configurator window is built. Definitely a learning curve here :smile: